release/models/aft/openconfig-aft-common.yang

```diff
@@ -303,7 +303,7 @@ submodule openconfig-aft-common {
             }
 
             container gre {
-              when "../state/type = 'oc-aftt:GRE'";
+              when "../state/type = 'oc-types:GRE'";
```

Comment:
Since there is a relocation of types to a separate model - the `openconfig-version` and `revision` must be updated to reflect this backwards incompatible change

Reply:
+1, but I don't see why we need to make this change. Why not leave these in AFT types?

Reply:
The next PR from @danameme  will use them in policy-forwarding

Reply:
It's used in policy-forwarding in this same PR.